### ISSUE TITLE
Clarification on definition of "scrolling in two dimensions" in 1.4.10

### ISSUE THREAD

**Issue by mra11yx** [w3c/wcag#2119]
Based on my reading of the 1.4.10 Understanding document and some of the issue requests raised here, I'm unclear on whether "scrolling in two dimensions" means:
1. The page should not require users to scroll in two dimensions **at the same time** to read content. Example: When you zoom to 400%, the page continues to have the same dimensions as it does at 100%, requiring the user to pan around, like they would have to do in a (traditionally) zoomed PDF.
**or**
2. The page should not require **any** kind of horizontal scrolling at 400% zoom, outside of the "required for understanding" exceptions, even if the user **doesn't** have to scroll up/down and left/right at the same time. Example: A menu bar in which the user needs to scroll left and right to perceive some of the items, but not up and down.

**Comment by JAWS-test** [comment]
My understanding of 1.4.10: Your option 1 is correct, however instead of "read content" there should be "perceive content" since "read" refers to text. But it can also be non-text content, e.g. 100 icons in a toolbar:
* Correct would be that this toolbar must be scrolled either vertically or horizontally only. 
* Not correct would be if the toolbar has to be scrolled vertically and horizontally.

**Comment by LaurenceRLewis** [comment]
Is there a reason this is still open? 

Is the meaning of "scrolling in two dimensions" accurately described as "_The page should not require users to scroll in two dimensions at the same time to perceive content._"

Does scrolling in two dimensions include the browser page level vertical, or equivalent horizontal scrolling when considered in  "scrolling in two dimensions"?

Do the ARIA TABS 1.2 fail WCAG 1.4.10 Reflow? #2284 - [The intent of the standard is that there should be either a horizontal or vertical scroll bar **at the browser page level**](https://github.com/w3c/aria-practices/issues/2284#:~:text=The%20intent%20of%20the%20standard%20is%20that%20there%20should%20be%20either%20a%20horizontal%20or%20vertical%20scroll%20bar%20at%20the%20browser%20page%20level)

Could we get clarification on this from the ARIA WG and close as concluded, or further discuss to achieve understanding.

**Comment by alastc** [comment]
We're working on an update for the Reflow understanding that should tackle a few of these issues.

@mra11yx I think option 2 is the closest to the intent, i.e. when you zoom-in the page reflows to keep things within the viewport. (Or in PDF, you use the reflow feature.)

@LaurenceRLewis In the ARIA tabs example, I think it's the content that's creating the scrolling?

However, assuming the tabs items across the top were wider than 320px there are several options:

- Wrapping the tabs (not a good or recommended option).
- Horizontally scrolling, with buttons for next/previous like a carousel.
- Horizontally scrolling without buttons, but (ideally) with a scroll bar shown.

So long as the tabs are less then 256px high, and don't cause full-page horizontal scrolling, they can scroll horizontally (assuming a vertically scrolling page).

**Comment by LaurenceRLewis** [comment]
@alastc "In the ARIA tabs example, I think it's the content that's creating the scrolling". 

I agree.

**Comment by scottaohara** [comment]
closing as resolved by the updated Reflow understanding doc merged in #4055 which will be published in april (for now [preview link for the updated understanding doc](https://deploy-preview-4055--wcag2.netlify.app/understanding/reflow))
